commit a32c5ff75254ded2b3148eefa315b7bbfb74443d
Author: Fred Emmott <fred@fredemmott.co.uk>
Date:   Tue Mar 27 08:23:26 2018 -0700

    Portability: `unsigned long` instead of `ulong`
    
    `ulong` is undefiend on macos, and otherwise unused. Blocks merging updates into HHVM.

diff --git a/third-party/webscalesqlclient/mysql-5.6/include/mysql.h b/third-party/webscalesqlclient/mysql-5.6/include/mysql.h
index 5aa89f4d58..41b1a792c3 100644
--- a/third-party/webscalesqlclient/mysql-5.6/include/mysql.h
+++ b/third-party/webscalesqlclient/mysql-5.6/include/mysql.h
@@ -670,8 +670,8 @@ unsigned int	STDCALL mysql_thread_safe(void);
 my_bool		STDCALL mysql_embedded(void);
 my_bool         STDCALL mysql_read_query_result(MYSQL *mysql);
 int             STDCALL mysql_reset_connection(MYSQL *mysql);
-ulong STDCALL cli_safe_read(MYSQL *mysql, my_bool *is_data_packet);
-net_async_status STDCALL cli_safe_read_nonblocking(MYSQL *mysql, ulong* res,
+unsigned long   STDCALL cli_safe_read(MYSQL *mysql, my_bool *is_data_packet);
+net_async_status STDCALL cli_safe_read_nonblocking(MYSQL *mysql, unsigned long *res,
                                                    my_bool *is_data_packet);
 
 
